MREUnityRuntime/MREUnityRuntimeLib/IPC/IConnection.cs

```diff
@@ -87,6 +88,8 @@ public interface IConnection : IDisposable
 		/// </summary>
 		/// <param name="message"></param>
 		void Send(string message);
```

Comment:
If IConnections are not operating on strings anymore, probably better to remove this overload.